### PR TITLE
Alex dashboard revamp

### PR DESCRIPTION
**Pull request by AlexBishop01**
CURRENT ADDITIONS

- User dashboard designed changed
- User will now get the number of events they have scheduled in their calendar

FUTURE ADDITIONS PREPARED BY CHANGES IN THIS PULL REQUEST

- User will be prompted by a "Welcome, ${username}" message with their account's username
- User will get the number of notes they have on the dashboard

### CODE REVIEW

**Review by kaylebredner — APPROVED** [review]
looks good to me

**Review by kaylebredner — COMMENTED** [review]
don't forget to call the functions from the html

File: pages/dashboard.html [hunk 1]
```diff
@@ -10,67 +10,79 @@
...
+                    <a class="nav-link active" href="/dashboard">Home</a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link active" href="/calendar">Calendar</a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link active" href="/notebook">Notebooks</a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link active" href="/settings">Settings</a>
```
Comment (tarunJeevan) [inline pages/dashboard.html:33]: I thought we decided to get rid of the Settings?

**Review by tarunJeevan — APPROVED** [review]
Looks good

File: pages/dashboard.html (hunk 1 above)
Comment (AlexBishop01) [reply pages/dashboard.html:33]: Yep, I just forgot to include it in this pull request. It'll be in the next one when I get the unit testing going later tonight

**Review by maitmd — APPROVED** [review]
Viewed on Discord looks good!

**Pull request merged**